__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require('../app')
+const request = require('supertest')
+const db= require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+
+beforeEach(()=>seed(data))
+afterAll(()=>db.end())
+
+describe('/api', () => {
```

Comment:
it might make sense to update this describe as we aren't accessing /api directly to get this 404! :)